test/simulator/timestamp/src/timestamp_manager.cpp

```diff
@@ -45,6 +45,30 @@ timestamp_manager::get_timestamp_manager()
     return (_timestamp_manager_instance);
 }
 
+uint64_t
+timestamp_manager::hex_to_decimal(const std::string &hex_ts)
+{
+    std::stringstream stream;
+    uint64_t ts;
+
+    stream << hex_ts;
+    stream >> std::hex >> ts;
+
+    return (ts);
+}
+
+std::string
+timestamp_manager::decimal_to_hex(const u_int64_t ts)
+{
+    std::stringstream stream;
+    std::string hex_ts;
```

Comment:
This one is not necessary, you can return `stream.str()`

Reply:
I did this for the easier readability of the code (so that the reader understands that the timestamp is changed from ts to hex_ts)

Reply:
I think it's fine to remove it

Reply:
I will remove it, thanks. Updated in [071beb2](https://github.com/wiredtiger/wiredtiger/pull/8167/commits/071beb29c2a76de924fb872cd47b5edd149962bf)